resources/groundwork/web.md

```diff
@@ -53,7 +53,38 @@ TODO
 
 ### Sync Stack
 
-TODO
+| Pros | Cons |
+|------|------|
+| Both Frontend and Backend are in one language (TypeScript). Express (Node.js) is low level, meaning it gives you a lot of control, allows for customizability, and is generally easy to debug. | React is not the easiest frontend framework to learn |
```

Comment:
I mentioned this in #131, but I think we can omit specific technologies in this overview - it should be covered in more detail further down :) Maybe something high-level like "shared language means easier onboarding for inexperienced members"